sports/.golangci.yml

```diff
@@ -0,0 +1,37 @@
+run:
```

Comment:
A lot of the build scripts, config and even code were just duplicated across for this one. It'd be better to have these sorts of things shared if they are to be identical.